CONTRIBUTING.md

```diff
@@ -7,6 +7,10 @@ Before you begin:
 - Have you read the [code of conduct](CODE_OF_CONDUCT.md)?
 - Check out the [existing issues](https://github.com/unleash/Unleash/issues)
 - Browse the [developer-guide](./website/docs/contributing/developer-guide.md) for tips on environment setup, running the tests, and running Unleash from source.
+- You need 
+  - Node 20
+  - corepack enabled `corepack enable`
+  - yarn installed globally `npm install -g yarn`
```

Comment:
```suggestion

```

I don't think this is necessary, the docs only say to enable corepack
![image](https://github.com/Unleash/unleash/assets/107407814/42a8cf5b-72e6-45d6-8a61-f2501ec09504)